tests/cpu_bugs/login_console.pm

```diff
@@ -0,0 +1,45 @@
+# SUSE's openQA tests
+#
+# Copyright © 2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+#
+#
+# Summary:  ssh login
```

Comment:
more details please. I don't see how "ssh login" relates to "login_console.pm". Overall I think this whole test module should not be needed.

Reply:
Yes, sorry, this module should be removed. it's garbage.